examples/02-echo/Cargo.toml

```diff
@@ -9,5 +9,5 @@ publish = false
 
 [dependencies]
 fluvio = { path = "../../src/client" }
-futures-lite = "1.10.1"
+futures = "0.3.9"
```

Comment:
is there reason putting 0.3.9? should put lowest bound much as possible

Reply:
Why lowest? Don't you typically choose latest version for dependencies?

Reply:
No, unless you have explicit requirements for that release (bug fixes, new API,etc).   If you put higher version number, you are imposing more requirements for user of create which force them to upgrade.